go/vt/orchestrator/inst/analysis_dao.go

```diff
@@ -58,6 +59,7 @@ func initializeAnalysisDaoPostConfiguration() {
 type clusterAnalysis struct {
 	hasClusterwideAction bool
 	primaryKey           *InstanceKey
+	durability           reparentutil.Durabler
```

Comment:
Durability is stored at the cluster level to avoid creating a new one for all the tablets of a shard. This could move to a keyspace level, but we don't have keyspace level common things yet.